.travis.yml

```diff
@@ -47,8 +51,17 @@ matrix:
   fast_finish: true
   include:
     # Builds that are executed for every PR
-    - os: osx # run base tests on both platforms
     - os: linux
+    # i686 toolchain could run on x86_64 system.
+    - os: linux
+      env: HOST_TOOLCHAIN=i686-unknown-linux-gnu
+      addons:
+        apt:
+          packages:
+            - gcc-multilib
+      # if: branch IN (auto, try)
```

Comment:
I guess this is WIP, to test it in this PR?

Reply:
yes, it is.